test/github-issues/3857/entity/Men.ts

```diff
@@ -0,0 +1,9 @@
+import {Person} from "./Person";
+import {ChildEntity, Column} from "../../../../src";
+
+@ChildEntity()
+export class Men extends Person {
+
+    @Column("bigint")
+    dickLength: number;
```

Comment:
can you change it together with boobs, I don't really want codebase to contain dicks.